### DISCUSSION TITLE
allure-cypress not automatically generate annotation of each test step

### DISCUSSION THREAD

**Discussion by lynnismetw** [allure-framework/allure2#2508] [Questions & Support] [answered]
As PieterGrobler mentioned in https://github.com/cypress-io/cypress/issues/27113#issuecomment-2033129596, allure-cypress lost the automatic annotation of each test step which we got from cypress-allure-plugin.

My project just upgraded Cypress to 13 and also replaced **cypress-allure-plugin** with **allure-cypress** but found NO test body with each step automatically generated in the allure result. Will you consider add this feature to allure-cypress?

**Comment by baev** [comment]
This issue is planned for the next iteration (starting April 29).

**Reply by lynnismetw** [reply]
Hi @baev, may I have the issue number so I may know the status better?

**Comment by kwangecs** [comment]
Hi @baev,
Are there any updates for this issue?

**Comment by baev** [comment, marked as answer]
fixed in `allure-cypress@3.0.0-beta.5`
